ou_dedetai/system.py

```diff
@@ -275,6 +276,7 @@ def tl(library):
 
 
 def get_dialog():
+    # FIXME: wouldn't the cli still work without a DISPLAY? What about wayland?
```

Comment:
This is about when executing the wine functions that require a GUI, not necessarily our CLI. We'd need to make this specified based on the CLI command run.